### ISSUE TITLE
Limit download size for user-input URLs

### ISSUE THREAD

**Issue by rgaudin** [openzim/zimit-frontend#67]
With illustration already and behaviors soon (maybe?), our worker can be ordered to download arbitrary URLs without stopping at any specific size. That's a potential hazard.

We should check how the crawler enforces the limit we use as well ; it could be a simple post-write check which might be vulnerable as well.

**Comment by benoit74** [comment]
Agreed, this is a "significant" security concern. Probably something to solve in scraper (or scraperlib even).
